PuzzleSlider/src/Engine/GameData.java

```diff
@@ -40,6 +44,10 @@ public String toString() {
 			out.append( Arrays.toString(mTiles[i]));
 			out.append(System.lineSeparator());
 		}
+		out.append("Pocet tahov: " + mMoveCount);
```

Comment:
Is there any reason for being inconsistent here? I mean, we are using english?. And, I hate java but is this really suitable in java to add `integer` to `string` whitout any formating function or sth like that?